mysql-test/suite/innodb/r/parallel_dblwr_encrypt.result

```diff
@@ -0,0 +1,59 @@
+SET GLOBAL innodb_page_cleaner_disabled_debug=ON;
```

Comment:
percona_ prefix for the Percona feature testcases

Reply:
Fixed